docs/rum_collection.md

```diff
@@ -0,0 +1,102 @@
+# Android Trace Collection
+
+<div class="alert alert-info">The Android trace collection is in public beta, and is currently not supported by Datadog.</div>
+
+Send [Real User Monitoring data][1] to Datadog from your Android applications with [Datadog's `dd-sdk-android` client-side RUM library][2] and leverage the following features:
+
+* get a global idea about your app’s performance and demographics;
+* understand which resources are the slowest;
+* analyze errors by OS and device type.
+
+**Note**: RUM on Android is still experimental, and will be available in the `dd-sdk-android` library version `1.5.0` or higher. The `dd-sdk-android` library supports all Android versions from API level 19 (Kit-Kat).
+
+## Setup
+
+1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
+
+    ```conf
+    repositories {
+        maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+    }
+
+    dependencies {
+        implementation "com.datadoghq:dd-sdk-android:x.x.x"
+    }
+    ```
+
+2. Initialize the library with your application context and your [Datadog client token][4]. For security reasons, you must use a client token: you cannot use [Datadog API keys][5] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][4]. You also need to provide an Application ID (see our [RUM Getting Started page][7]).
+
+    {{< tabs >}}
+    {{% tab "US" %}}
+
+```kotlin
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val config = DatadogConfig.Builder(BuildConfig.DD_CLIENT_TOKEN, BuildConfig.APPLICATION_ID)
+                        .trackGestures()
+                        .useViewTrackingStrategy(strategy)
+                        .build()
+        Datadog.initialize(this, config)
+    }
+}
+```
+
+    {{% /tab %}}
+    {{% tab "EU" %}}
+
+```kotlin
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val config = DatadogConfig.Builder(BuildConfig.DD_CLIENT_TOKEN, BuildConfig.APPLICATION_ID)
```

Comment:
now that we are making the "environment" mandatory I think we should add it here already. Please check my PR later today.